src/main/java/net/pms/util/StringUtil.java

```diff
@@ -41,6 +41,18 @@
 	public static final String SEC_TIME_FORMAT = "%02d:%02d:%02.0f";
 	public static final String DURATION_TIME_FORMAT = "%02d:%02d:%05.2f";
 	public static final String NEWLINE_CHARACTER = System.getProperty("line.separator");
+	public static final long KIBI = 1L << 10;
```

Comment:
I agree with that unit choice for memory.  The SI units are only used for Network bandwidth and disk size. IEC units are used everywhere else.  [Ubuntu published a UnitsPolicy in 2010](https://wiki.ubuntu.com/UnitsPolicy) that's generally consistent across Linux distributions.

> Use base-10 for:
> 
> network bandwidth (for example, 6 Mbit/s or 50 kB/s)
> disk sizes (for example, 500 GB hard drive or 4.7 GB DVD)
> Use base-2 for:
> 
> RAM sizes (for example, 2 GiB RAM)
> For file sizes there are two possibilities:
> 
> Show both, base-10 and base-2 (in this order). An example is the Linux kernel: "2930277168 512-byte hardware sectors: (1.50 TB/1.36 TiB)"
> Only show base-10, or give the user the opportunity to decide between base-10 and base-2 (the default must be base-10).

You may see memory marketed as 16 GB, but rest assured that the memory chips actually contain 16 GiB.